### ISSUE TITLE
ImageNet: changing batch-size affects point when lr switches

### ISSUE THREAD

**Issue by yaroslavvb** [diux-dev/cluster#41]
This is either unwanted interaction, or measurement side-effect (right now x-axis doesn't properly count incomplete batches, this line https://github.com/diux-dev/cluster/blob/13fb813d4a3a96a6c27bda1d5bee979d0d3d125e/pytorch/training/train_imagenet_nv.py#L466) . Ideally it should be possible to change batch size without affecting other parameters like schedule

cc @bearpelican 

[tb link](
http://35.174.18.63:6006/#scalars&tagFilter=times%2Fstep&runSelectionState=eyJlaWdodC1tYWNoaW5lcy12YSI6ZmFsc2UsIm9uZS1tYWNoaW5lcy12YSI6ZmFsc2UsImVpZ2h0LW1hY2hpbmVzLXZhMDIiOmZhbHNlLCJuY2NsLWVpZ2h0LW1hY2hpbmVzLWFyIjpmYWxzZSwibmNjbC1laWdodC1tYWNoaW5lcy1hci4yMDE4LTA3LTMwXzIzLTI0LTE0IjpmYWxzZSwib25lLW1hY2hpbmVzMTI4IjpmYWxzZSwiZGVsZXRlbWUiOmZhbHNlLCJ3ZWQtdHdvLW1hY2hpbmVzIjpmYWxzZSwiZGVsZXRlbWUuMDEiOmZhbHNlLCJ3ZWQtdHdvLW1hY2hpbmVzLjAxIjpmYWxzZSwicGlsbG93LW9uZSI6dHJ1ZSwicGlsbG93LXR3byI6dHJ1ZX0%3D)

<img width="524" alt="screenshot 2018-08-01 19 30 05" src="https://user-images.githubusercontent.com/23068/43559078-50f98ed6-95c1-11e8-80b7-57a4312bd2d6.png">

**Comment by bearpelican** [comment]
Oh this drop in learning rate is actually intentional.
For some reason I found that when setting batch norm weights to 0, we get better results scaling to 2x learning rate before dropping back down.
https://github.com/diux-dev/cluster/blob/master/pytorch/training/train_imagenet_nv.py#L187

Will need to do more tests to understand why this is so. 
But to do the traditional warmup, with no drop, disable the '--init-bn0' flag
